x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/rule/RuleExecutor.java

```diff
@@ -128,43 +125,17 @@ public TreeType after() {
             return transformations;
         }
     }
-
-    protected TreeType executeWithMetrics(TreeType plan, Map<String, CounterMetric> featuresMetrics) {
-        return execute(plan, featuresMetrics);
-    }
     
     protected TreeType execute(TreeType plan) {
-        return execute(plan, null);
-    }
-    
-    private TreeType execute(TreeType plan, Map<String, CounterMetric> featuresMetrics) {
-        return executeWithInfo(plan, featuresMetrics).after;
-    }
-    
-    protected ExecutionInfo executeWithInfo(TreeType plan) {
-        return executeWithInfo(plan, null);
-    }
-    
-    protected ExecutionInfo executeWithInfoWithMetrics(TreeType plan, Map<String, CounterMetric> featuresMetrics) {
-        return executeWithInfo(plan, featuresMetrics);
+        return executeWithInfo(plan).after;
```

Comment:
Has the file been reverted to the original form? If that happens I would expect this file (and others) to not show up anymore.